main.go

```diff
@@ -228,7 +253,7 @@ func init() {
 	corectlCommand.PersistentFlags().String("engine-headers", "30", "HTTP headers to send to the engine")
 	viper.BindPFlag("engine-headers", corectlCommand.PersistentFlags().Lookup("engine-headers"))
 
-	corectlCommand.PersistentFlags().StringP("app", "a", "unnamed-app.qvf", "App name including .qvf file ending")
+	corectlCommand.PersistentFlags().StringP("app", "a", "", "App name including .qvf file ending")
```

Comment:
I think this part needs to be documented in the README as well, since the documentation is quite unclear right now

Reply:
Done